examples/abitest/module_0/rust/src/lib.rs

```diff
@@ -2153,6 +2190,78 @@ impl FrontendNode {
         Ok(())
     }
 
+    fn test_http_server_create(&mut self) -> TestResult {
+        // Create an HTTP server pseudo-Node.
+        let result = oak::http::init("[::]:8282");
+        expect_matches!(result, Ok(_));
+        let invocation_receiver = result.unwrap();
+        // Close the only read-handle for the invocation handle, which should
+        // trigger the HTTP server pseudo-Node to terminate (but we can't
+        // check that here).
+        expect_eq!(Ok(()), invocation_receiver.close());
+        Ok(())
+    }
+
+    fn test_http_server_invalid_port(&mut self) -> TestResult {
+        // Attempt to create an HTTP server with an invalid local port.
```

Comment:
FWIW, I don't think the port is invalid, it just requires higher privileges to bind to.